### PR TITLE
Update - Protect endpoint to bn_boost, bump versions of core, react and vue

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
Update the bn_protect endpoint to bn_boost(private rpc aggregator), bump versions of core, react and vue

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### CODE REVIEW

File: docs/src/routes/docs/[...3]modules/[...1]core/+page.md [hunk 1]
```diff
@@ -164,6 +164,7 @@ type Chain = {
   publicRpcUrl?: string // an optional public RPC used when adding a new chain config to the wallet
   blockExplorerUrl?: string // also used when adding a new config to the wallet
   secondaryTokens?: SecondaryTokens[] // An optional array of tokens (max of 5) to be available to the dapp in the app state object per wallet within the wallet account and displayed in Account Center (if enabled)
+  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private and protected RPC to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...3]modules/[...1]core/+page.md:167]:
```suggestion
  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private RPC aggregator to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions. More information can be founr at `https://docs.blocknative.com/blocknative-mev-protection/blocknative-protect-rpc-endpoint`
```

File: packages/core/README.md [hunk 2]
```diff
@@ -148,6 +148,7 @@ type Chain = {
   publicRpcUrl?: string // an optional public RPC used when adding a new chain config to the wallet
   blockExplorerUrl?: string // also used when adding a new config to the wallet
   secondaryTokens?: SecondaryTokens[] // An optional array of tokens (max of 5) to be available to the dapp in the app state object per wallet within the wallet account and displayed in Account Center (if enabled)
+  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private and protected RPC to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions
```
Comment (Adamj1232) [inline packages/core/README.md:151]:
```suggestion
  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private RPC aggregator to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions. More information can be founr at `https://docs.blocknative.com/blocknative-mev-protection/blocknative-protect-rpc-endpoint`
```

File: docs/src/routes/docs/[...3]modules/[...1]core/+page.md [hunk 3]
```diff
@@ -164,6 +164,7 @@ type Chain = {
   publicRpcUrl?: string // an optional public RPC used when adding a new chain config to the wallet
   blockExplorerUrl?: string // also used when adding a new config to the wallet
   secondaryTokens?: SecondaryTokens[] // An optional array of tokens (max of 5) to be available to the dapp in the app state object per wallet within the wallet account and displayed in Account Center (if enabled)
+  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private RPC aggregator to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions. More information can be founr at `https://docs.blocknative.com/blocknative-mev-protection/transaction-boost-alpha`
```
Comment (taylorjdawson) [inline docs/src/routes/docs/[...3]modules/[...1]core/+page.md:167]:
```suggestion
  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private RPC aggregator to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions. More information can be found at `https://docs.blocknative.com/blocknative-mev-protection/transaction-boost-alpha`
```

File: packages/core/README.md [hunk 4]
```diff
@@ -363,6 +364,20 @@ type AccountCenter = {
   position?: AccountCenterPosition // default: 'bottomRight'
   expanded?: boolean // default: true
   minimal?: boolean // enabled by default for mobile
+  /**
+   * false by default - This allows removal of the
+   * Enable Transaction Protection' button within the Account Center
+   * expanded when set to true
+   * Can be set as a global for Account Center or per interface (desktop/mobile)
+   */
+  hideTransactionProtectionBtn?: boolean
```
Comment (taylorjdawson) [inline packages/core/README.md:373]:
```suggestion
/**
 * Controls the visibility of the 'Enable Transaction Protection' button within the expanded Account Center.
 * - When set to false (default), the button is visible.
 * - When set to true, the button is hidden.
 * This setting can be configured globally for the Account Center, or separately for different interfaces like desktop/mobile.
 */
```

**Review by taylorjdawson — APPROVED** [review]
looks good! just some small comments

File: docs/src/routes/docs/[...3]modules/[...1]core/+page.md (hunk 3 above)
Comment (Adamj1232) [reply docs/src/routes/docs/[...3]modules/[...1]core/+page.md:167]: ahhh good catch!

File: packages/core/README.md [hunk 5]
```diff
@@ -148,6 +148,7 @@ type Chain = {
   publicRpcUrl?: string // an optional public RPC used when adding a new chain config to the wallet
   blockExplorerUrl?: string // also used when adding a new config to the wallet
   secondaryTokens?: SecondaryTokens[] // An optional array of tokens (max of 5) to be available to the dapp in the app state object per wallet within the wallet account and displayed in Account Center (if enabled)
+  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private RPC aggregator to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions. More information can be founr at `https://docs.blocknative.com/blocknative-mev-protection/transaction-boost-alpha`
```
Comment (Adamj1232) [inline packages/core/README.md:151]:
```suggestion
  protectedRpcUrl?: string //An optional protected RPC URL - Defaults to Blocknative's private RPC aggregator to allow users to update the chain RPC within their wallet, specifically for private RPCs that protect user transactions. More information can be found at `https://docs.blocknative.com/blocknative-mev-protection/transaction-boost-alpha`
```

**Pull request merged**